recipes/mdns/all/test_v1_package/test_package.c

```diff
@@ -0,0 +1,43 @@
+
+#ifdef _WIN32
```

Comment:
Can we simplify this a bit? No need to (try to) open sockets in the test packages

Reply:
Sure. I can add it next week. And this test_v1_package should actually be removed, since it's not actually used by CMakeLists.txt.